apps/api/src/app/inbox/inbox.controller.ts

```diff
@@ -432,6 +458,31 @@ export class InboxController {
     );
   }
 
+  @UseGuards(AuthGuard('subscriberJwt'))
+  @Patch('/preferences/subscriptions/:subscriptionIdOrIdentifier/workflows/:workflowIdOrIdentifier')
```

Comment:
if this route is not restful anyway, i wonder if we should make the route shorter

```suggestion
  @Patch('/preferences/:subscriptionIdOrIdentifier/:workflowIdOrIdentifier')
```


as even today the patch preference is `/preferences/:workflowIdOrIdentifier` and not `/preferences/workflows/:workflowIdOrIdentifier`

Reply:
done